pom.xml

```diff
@@ -245,7 +245,8 @@
         <version>${maven-surefire-plugin.version}</version>
         <configuration>
           <skip>${skipUTs}</skip>
-          <argLine>-Xms256m -Xmx2g</argLine>
+          <argLine>-Xmx2g</argLine>
```

Comment:
this seems like change carried over from the other PR?

Reply:
Trying to improve the config as well...thinking -Xms it is not needed. And below this line, extending process exit timeout from 30 to 120 sec should be safe to do. I can also revert the `pom.xml` if you prefer separate the changes from this PR.

Reply:
its fine for now.. but we can split the PRs next time.. 

Also appreciate if you added a more descriptive commit message (we need to standardize this as a group)

Reply:
@vinothchandar noted. I'll make sure message is self-explanatory onwards.